src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
```

Comment:
We should just add a condition here called `'$(EnableDotnetAnalyzers)' == 'true'` and we should set it in the repo level dir.props to enable these. That way it only gets enabled for the repo's that want it.